src/utils/update/fetch.beneficial.owners.private.data.ts

```diff
@@ -4,7 +4,7 @@ import { ApplicationData } from "../../model";
 import { mapCorporateOrGovernmentBOPrivateData, mapIndividualBOPrivateData } from "./psc.to.beneficial.owner.type.mapper";
 
 export const fetchBeneficialOwnersPrivateData = async (appData: ApplicationData, req) => {
-
+  logger.info(`******** Fetching private beneficial owners data ********`);
```

Comment:
Do we need all the ******s in these messages? Suggest we remove them.

Reply:
They're only temp messages, but happy to remove the asterisks

Reply:
Fine if intention is to remove them, but then we should have a TODO comment stating this. I think just remove the *s. Up to you though.

Reply:
Removed the asterisks.